test/core/tsi/s2a/s2a_record_protocol_test.cc

```diff
@@ -741,8 +742,7 @@ static void s2a_test_decrypt_alert(uint16_t ciphersuite,
   grpc_core::Delete<grpc_channel>(channel);
 }
 
-static void s2a_test_random_roundtrips(uint16_t ciphersuite,
-                                       size_t iterations) {
+static void s2a_test_random_roundtrips(uint16_t ciphersuite) {
```

Comment:
Is there anything random in this test? Asking to see if we can omit the word "random" from the name.

Reply:
Good catch - removed "random".